src/python/pants/option/global_options.py

```diff
@@ -201,3 +201,5 @@ def register_options(cls, register):
     register('--subproject-roots', type=list, advanced=True, fromfile=True, default=[],
              help='Paths that correspond with build roots for any subproject that this '
                   'project depends on.')
+    register('--force', type=bool, recursive=True,
```

Comment:
I concur that this flag should also avoid the cache, i.e., actually force whatever it is you wanted to happen to happen. My example use-case is, I'm debugging a compiler warning (not error). So I need to keep re-invoking the compiler.